RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Details/NonFungibleTokenDetails+View.swift

```diff
@@ -279,9 +283,23 @@ private extension String {
 			}
 		}
 	}
+
+	var asInstantDataField: ArbitraryDataFieldKind? {
+		nilIfEmpty.map {
+			if let timeInterval = Int64($0) {
```

Comment:
Hmm this is an extension on `String`, so the i64 is sent as a String? That sounds strange and wrong?

Reply:
yes, it is returned as String as per GW contract. i64 refers to the [scrypto sbor value kind](https://github.com/radixdlt/babylon-wallet-ios/blob/216b24d52913544c4d471f656224f301112230ef/RadixWallet/Clients/GatewayAPI/CodeGen/Input/gateway-api-schema.yml#L3454), and we then validate the payload as actually being the Int64